personalize.module

```diff
@@ -1100,6 +1100,17 @@ function personalize_agent_set_status($agent_name, $status) {
     $end_time_var = _personalize_agent_get_stoptime_variable($agent_name);
     $now = time();
     variable_set($end_time_var, $now);
+    // Check if we need to delete all the actions for completed campaigns.
```

Comment:
Should this also happen when a campaign is deleted via personalize_agent_delete?  We delete the goals there directly in the db.